development/mimir-microservices-mode/config/grafana-agent.yaml

```diff
@@ -11,54 +11,34 @@ prometheus:
     - name: local
       host_filter: false
       scrape_configs:
-        - job_name: mimir-microservices-mode/distributor
-          static_configs:
-            - targets: ['distributor-1:8000', 'distributor-2:8001']
-              labels:
-                cluster: 'docker-compose'
-                namespace: 'mimir-microservices-mode'
-        - job_name: mimir-microservices-mode/ingester
-          static_configs:
-            - targets: ['ingester-1:8002', 'ingester-2:8003']
-              labels:
-                cluster: 'docker-compose'
-                namespace: 'mimir-microservices-mode'
-        - job_name: mimir-microservices-mode/querier
-          static_configs:
-            - targets: ['querier:8004']
-              labels:
-                cluster: 'docker-compose'
-                namespace: 'mimir-microservices-mode'
-        - job_name: mimir-microservices-mode/ruler
-          static_configs:
-            - targets: ['ruler-1:8021', 'ruler-2:8022']
-              labels:
-                cluster: 'docker-compose'
-                namespace: 'mimir-microservices-mode'
-        - job_name: mimir-microservices-mode/compactor
-          static_configs:
-            - targets: ['compactor:8006']
-              labels:
-                cluster: 'docker-compose'
-                namespace: 'mimir-microservices-mode'
-        - job_name: mimir-microservices-mode/query-frontend
-          static_configs:
-            - targets: ['query-frontend:8007']
-              labels:
-                cluster: 'docker-compose'
-                namespace: 'mimir-microservices-mode'
-        - job_name: mimir-microservices-mode/store-gateway
-          static_configs:
-            - targets: ['store-gateway-1:8008', 'store-gateway-2:8009']
-              labels:
-                cluster: 'docker-compose'
-                namespace: 'mimir-microservices-mode'
-        - job_name: mimir-microservices-mode/query-scheduler
-          static_configs:
-            - targets: ['query-scheduler:8011']
-              labels:
-                cluster: 'docker-compose'
-                namespace: 'mimir-microservices-mode'
+        - job_name: mimir-microservices
+          static_configs:
+            - targets:
+                - 'distributor-1:8000'
+                - 'distributor-2:8001'
+                - 'ingester-1:8002'
+                - 'ingester-2:8003'
+                - 'querier:8004'
+                - 'ruler-1:8021'
+                - 'ruler-2:8022'
+                - 'compactor:8006'
+                - 'query-frontend:8007'
+                - 'store-gateway-1:8008'
+                - 'store-gateway-2:8009'
+                - 'query-scheduler:8011'
+              labels:
+                cluster: 'docker-compose'
+                namespace: 'mimir-microservices-mode'
```

Comment:
Can we make `namespace` and `job` prefix (before `/`) match?

Reply:
My bad, this is already correct, as `job` is computed from namespace, not `job_name` from this config file.